README.md

```diff
@@ -32,33 +32,23 @@
 - [❓ FAQ](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
 # 📖 [Math-Magician]
 
-> Describe your project in 1 or 2 sentences.
-
 **[Math Magician]** 
 -This project is math magician for calculating figures
 
 ## 🛠 Built With
 -React
 
-<!-- Features -->
 
 ### Key Features
 
-> Describe between 1-3 key features of the application.
-
 - **[key_feature_1]**
```

Comment:
I'd suggest you add a feature here instead of the [key_feature_1]. It will make your README file look more professional 😉